package.json

```diff
@@ -49,7 +49,7 @@
     "tsup": "^8.0.2",
     "turbo": "^1.13.3",
     "typescript": "^4.7.4",
-    "vitest": "^0.6.3",
+    "vitest": "0.6.3",
```

Comment:
These were getting resolved to a much newer version due to the new dependencies so needed to pin them to prevent breakage..